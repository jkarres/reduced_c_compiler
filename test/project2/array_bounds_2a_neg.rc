// writing to an invalid location (negative)

function : void main() {
  int[9] ia;

  int i = -1;

  ia[i] = i;

}
