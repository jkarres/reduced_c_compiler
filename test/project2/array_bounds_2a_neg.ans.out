Index value of -1 is outside legal range [0,9).
